debug.js

```diff
@@ -0,0 +1,2 @@
+// no-op replacement for npm.im/debug to optimize it away.
+export default () => {};
```

Comment:
debug does have browser support via localstorage. it sounds like this would remove that support. I don't think people use this feature though, so I think it's fine.